ispyb-ejb/src/main/java/ispyb/common/util/Constants.java

```diff
@@ -375,6 +375,10 @@ public static final boolean IS_INDUSTRY_MAILING_IN_PROD() {
 
 	public static final String[] DEWAR_AUTOMATIC_PROCESSING_STATE = { DEWAR_ID30 };
 
+	public static final String ARRIVAL_LOCATION = "STORES-IN";
+
+	public static final String RETURN_LOCATION = "STORES-OUT";
```

Comment:
There are already such constants : 

String DEWAR_STORES_IN = "STORES-IN";
String DEWAR_STORES_OUT = "STORES-OUT";

Could you please use these ones ?

Reply:
It was a mistake.